File: src/main/java/br/com/correios/enderecador/cep/gpbe/GpbeStrategyCEP.java
package br.com.correios.enderecador.cep.gpbe;

import br.com.correios.enderecador.cep.CEPFactory;
import br.com.correios.enderecador.cep.StrategyCEP;

public class GpbeStrategyCEP implements StrategyCEP {
    public CEPFactory getFactory() {
        return CEPFactory.factory(2);
    }
}

File: src/main/kotlin/br/com/correios/enderecador/cep/gpbe/GpbeStrategyCEP.kt
package br.com.correios.enderecador.cep.gpbe

import br.com.correios.enderecador.cep.CEPFactory
import br.com.correios.enderecador.cep.StrategyCEP

class GpbeStrategyCEP : StrategyCEP {
    override val factory: CEPFactory?
        get() = CEPFactory.factory(2)
}
